modules/core/src/main/scala/org/scalasteward/core/buildtool/maven/MavenAlg.scala

```diff
@@ -37,19 +37,29 @@ object MavenAlg {
       F: Monad[F]
   ): MavenAlg[F] =
     new MavenAlg[F] {
-      override def containsBuild(repo: Repo): F[Boolean] =
-        workspaceAlg.repoDir(repo).flatMap(repoDir => fileAlg.isRegularFile(repoDir / "pom.xml"))
+      override def containsBuild(buildRoot: BuildRoot): F[Boolean] =
+        workspaceAlg
+          .repoDir(buildRoot.repo)
+          .flatMap(repoDir =>
+            fileAlg.isRegularFile(repoDir / buildRoot.relativeBuildRootPath / "pom.xml")
+          )
```

Comment:
@GrafBlutwurst Could you add a new `buildDir` or `buildRootDir` method to `WorkspaceAlg` that is implemented like this
```scala
def buildDir(buildRoot: BuildRoot): F[File] =
  repoDir(buildRoot.repo).map(_ / buildRoot.relativeBuildRootPath)
```
With that method, a lot of the changes in this PR become two simple replacements: `Repo` -> `BuildRoot` and `workspaceAlg.repoDir` -> `workspaceAlg.buildDir`.

Reply:
Done